e2e/playwright/snapshot-tests.spec.ts

```diff
@@ -1067,7 +1048,6 @@ test.describe('Grid visibility', { tag: '@snapshot' }, () => {
 })
 
 test('theme persists', async ({ page, context }) => {
```

Comment:
This one shouldn't be removed, as it's not a snapshot test per se (no @snapshot tag) and has been disabled for a long time. Electron tests don't have the || true espace

https://github.com/KittyCAD/modeling-app/commit/2c6404f6716f40f744afccffe227e75c70bec0dd#diff-380391eb09c3d1acda12bb524bc2ec78178f6378ae3ff32b5017983b6d9b83f0R1092

```suggestion
test('theme persists', async ({ page, context }) => {
  test.fixme(orRunWhenFullSuiteEnabled())
```